package.json

```diff
@@ -22,15 +22,15 @@
   "devDependencies": {
     "@stoplight/types": "4.x.x",
     "@types/chance": "1.0.x",
-    "@types/jest": "24.0.12",
-    "@types/node": "12.0.0",
+    "@types/jest": "24.x.x",
+    "@types/node": "12.x.x",
     "chance": "1.0.x",
-    "jest": "24.8.0",
+    "jest": "24.x.x",
     "lerna": "3.13.4",
     "ts-jest": "24.x.x",
     "tslint": "5.16.0",
-    "tslint-config-stoplight": "1.2.x",
```

Comment:
I notice that everyone has their own personal preference and they change them as they go. It concerns me that we change these so much, and that we do not seem to notice the technical differences behind them.

Pegging to specifics = not good, we miss security updates.
Using x = we are opening up the possibility of a dependency rolling back a dependency if it has strict requirements and we would only notice if we keep a close eye on the lock file

I generally prefer ^24.0.12 because that says "I'll take anything in the 24.0.x range so long as its over 12, because 11 had a bug that was messing us up and there is a reason I said ^24.0.12"

@chris-miaskowski you are one of the folks I see changing these things a lot too, would you agree with the above? :)

Reply:
My preference is: _whatever yarn puts_, I am just sticking to the rules people are giving me

Reply:
@philsturgeon I can go with either as long as we are consistent. `.x` was the way to go so far, I'm happy to move away from it.

Reply:
Sweeet, let's try and stick to ^ as it says "or newer", and if we do not care at all which minor version or patch is being used then do ^ and 0 instead of an x so its clear.

Reply:
Ok so shall I modify all the dependencies here now?

Reply:
Sorry, please. Then let’s merge this beast.

Reply:
Careting done

Reply:
8091c09